wiki/Manual.md

```diff
@@ -2848,6 +2849,25 @@ Returns
 }
 ```
 
+## Open Interest
+
+*contract only*
+
+Use the `fetchOpenInterest` method to get the open interest for a symbol from the exchange.
+
+```JavaScript
+fetchOpenInterest (symbol, params = {})
+```
+
+Parameters
+
+- **symbol** (String) Unified CCXT market symbol (e.g. `"BTC/USDT:USDT"`)
+- **params** (Dictionary) Extra parameters specific to the exchange API endpoint (e.g. `{"endTime": 1645807945000}`)
+
+Returns
+
+- A dictionary of [open interest structures](#open-interest-structure)
+
```

Comment:
Sorry, let's actually move this addition to a new PR, and let's implement `fetchOpenInterest` on several more exchanges before merging in the documentation, because it's get's more difficult to change the method after that

Reply:
Okay no problem it's already implemented on okx, huobi, aax and ftx, there's also a bitget PR pending

Reply:
https://github.com/ccxt/ccxt/pull/15254